dspace-api/src/main/java/org/dspace/core/LoggerServiceImpl.java

```diff
@@ -1,182 +0,0 @@
-/**
```

Comment:
This class is referenced in 'dspace-api/src/main/resources/spring/spring-dspace-core-services.xml'.  I think we can just remove the <bean> declaration there.